README.md

```diff
@@ -2,3 +2,8 @@ Sandbox for Game Engine experiments.
 
 ![](Docs/scene.png)
 ![](Docs/physics.gif)
+
+> Give someone a game engine and they deliver a game.  
+> Teach someone to make a game engine and they never deliver anything.  
```

Comment:
Could you be more elaborate here? I'd say appending "...not even the game engine" or similar makes things more clear.